src/sentry/static/sentry/app/views/projectsDashboard/projectCard.tsx

```diff
@@ -42,7 +49,7 @@ class ProjectCard extends React.Component {
     const {id, firstEvent, stats, slug} = project;
 
     return (
-      <ProjectCardWrapper data-test-id={slug} width={['100%', '50%', '33%', '25%']}>
```

Comment:
Was this width prop unused? It looks reflexbox related.

Reply:
Yes, this was reflexbox. Replaced with the css grid layout